.github/workflows/next.yml

```diff
@@ -0,0 +1,78 @@
+name: Pre-release
+
+on:
+  push:
+    branches:
+      - next
+
+jobs:
+  build-test:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+
+      - name: Cache node_modules
+        id: cache-modules
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-build-${{ hashFiles('yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.cache-modules.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+
+      - name: Build
+        run: yarn build
+
+      - name: Run tests
+        run: yarn test
+
+  prerelease:
+    name: Prerelease
+    if: github.ref == 'refs/heads/master'
```

Comment:
This doesn't seem right, I think? Wouldn't this not run the NPM release for the `next` branch?